ush/forecast_postdet.sh

```diff
@@ -217,6 +221,21 @@ EOF
         cpreq "${increment_file}" "${DATA}/INPUT/${inc_file}"
       done
 
+      # Land IAU increments: sfc_inc in FV3 grid, all timesteps in one file per tile 
+      if [[ ${DO_LAND_IAU} = ".true." ]]; then
+        local TN sfc_increment_file
+        for TN in $(seq 1 "${ntiles}"); do
+          sfc_increment_file="${COMIN_ATMOS_ANALYSIS}/sfc_inc.tile${TN}.nc"
+          if [[ ! -f "${sfc_increment_file}" ]]; then
+            echo "FATAL ERROR: DO_LAND_IAU=${DO_LAND_IAU}, but missing increment file ${sfc_increment_file}, ABORT!"
+            exit 1
+          fi
+          ${NCP} "${sfc_increment_file}" "${DATA}/INPUT/sfc_inc.tile${TN}.nc"
+        done
+        LAND_IAU_INC_FILES=${LAND_IAU_INC_FILES:-'sfc_inc'}
```

Comment:
```suggestion
        LAND_IAU_INC_FILES='sfc_inc'
```
At this time it is clear that the land IAU inc files are `sfc_inc`, as we just copied them in the above for loop.

Reply:
There is still a possibility that this will change. It will depend on the UFS_UTILS regridding issues that are yet to be sorted.